spec/system/consumer/shops_spec.rb

```diff
@@ -23,9 +23,40 @@
     producer.set_producer_property 'Organic', 'NASAA 12345'
   end
 
-  it "searches by URL" do
-    visit shops_path(anchor: "/?query=xyzzy")
-    expect(page).to have_content "Sorry, no results found for xyzzy"
+  context "searching enterprises" do
+    context "which exist" do
+      it "by URL" do
+        visit shops_path(anchor: "/?query=Enterprise")
+        expect(page).to have_content "Did you mean?"
+        expect(page).to have_content distributor.name
```

Comment:
Sure, this will work in the case that the query returns any matches.

If there are no results, then the UI will return something like: "Did you mean?" + "<query>", but this is not the case here.